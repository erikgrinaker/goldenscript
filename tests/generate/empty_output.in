# The parser should handle multiple empty output blocks in sequence, including
# an empty block at the end of the file.
command
---

command
---

command
---