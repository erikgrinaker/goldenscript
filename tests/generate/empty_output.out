# The parser should handle multiple empty output blocks in sequence, including
# an empty block at the end of the file.
command
---
Command { name: "command", args: [], prefix: None, silent: false, line_number: 3 }

command
---
Command { name: "command", args: [], prefix: None, silent: false, line_number: 6 }

command
---
Command { name: "command", args: [], prefix: None, silent: false, line_number: 9 }
